### PR TITLE
chore: update contract doc

### PR DESCRIPTION
**Pull request by PainterPuppets**
The description of some commands in the previous contract deployment pr has been changed and updated to the latest usage.

### CODE REVIEW

**Review by Daryl-L — APPROVED** [review]

**Comment by Keith-CY** [comment]
Based on the diffs I just found the options `--send` was removed, how does it break the workflow?

**Comment by Keith-CY** [comment]
Update: we are going to add an independent doc for how to deploy contracts and reference it in the MVP docs

**Comment by Keith-CY** [comment]
Any update on the new document of how to deploy contracts @PainterPuppets

**Comment by Keith-CY** [comment]
> Any update on the new document of how to deploy contracts @PainterPuppets

Repost @PainterPuppets

**Comment by PainterPuppets** [comment]
> > Any update on the new document of how to deploy contracts @PainterPuppets
> 
> Repost @PainterPuppets

Sorry I just saw it. It's been updated.

**Comment by Keith-CY** [comment]
It would be helpful for the demonstration to prepend the real-world contract `test-c` displayed in the document.

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# How to deploy contracts with kuai-cli
+
+## deploy contract workflow by builtin signer (directly deploy):
+
+1. `kuai contract new --name [contract-name]`
+   <img width="735" alt="image" src="https://github.com/ckb-js/kuai/assets/22258327/53d53673-b50b-426d-9604-2b8481ae6222">
+
+2. `kuai contract build --release`
+
+<img width="755" alt="image" src="https://github.com/ckb-js/kuai/assets/22258327/7e76b8e5-ac87-4eba-b3dc-aa233ad79fca">
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
   <img width="755" alt="image" src="https://github.com/ckb-js/kuai/assets/22258327/7e76b8e5-ac87-4eba-b3dc-aa233ad79fca"> 
```

**Comment by Keith-CY** [comment]
What if `copy-paste` commands and outputs in the terminal instead of screenshots. It would be easy to edit and copy to run

**Review by yanguoyu — APPROVED** [review]

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# How to deploy contracts with kuai-cli
+
+## deploy contract workflow by builtin signer (directly deploy):
+
+1. Create a new conrtact, skip this step if the contract already exists ([How to import exists contract](#how-to-import-contract-to-kuai-project))
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
1. Create a new conrtact, skip this step if the contract already exists ([How to import existent contract](#how-to-import-contract-to-kuai-project))
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+```bash
+$ kuai contract new --name always-success
+
+New contract "always-success"
+    Created binary (application) `always-success` package
+Rewrite Cargo.toml
+Rewrite capsule.toml
+Done
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  $ kuai contract new --name always-success
  
  # New contract "always-success"
  #     Created binary (application) `always-success` package
  # Rewrite Cargo.toml
  # Rewrite capsule.toml
  # Done
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+- `kuai contract build --name [contract-name] --release`
+
+```bash
+$ kuai contract build --name always-success --release
+
+Building contract always-success
+   Compiling always-success v0.1.0 (/code/contracts/always-success)
+Done
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  $ kuai contract build --name always-success --release
  
  # Building contract always-success
  #   Compiling always-success v0.1.0 (/code/contracts/always-success)
  # Done
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+```bash
+$ kuai contract deploy --name always-success --from ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --signer ckb-cli
+
+[warn] ANYONE_CAN_PAY script info not found in config!
+[warn] OMNILOCK script info not found in config!
+[warn] `config` changed, regenerate lockScriptInfos!
+The address format generated by generateAddress or scriptToAddress will be deprecated, please migrate to encodeToAddress to generate the new ckb2021 full format address as soon as possible
+Input ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq0rjr2tndx8vdlvspuehkhkgjhxyhxmjgsqw8hsk's password for sign messge by ckb-cli:
+deploy success, txHash:  0x7507b59b5f715c0fcd83d55df5569d72ff1cb05cd399afe1f366e03918827a58
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  $ kuai contract deploy --name always-success --from 
  ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --signer ckb-cli

  # [warn] ANYONE_CAN_PAY script info not found in config!
  # [warn] OMNILOCK script info not found in config!
  # [warn] `config` changed, regenerate lockScriptInfos!
  # The address format generated by generateAddress or scriptToAddress will be 
  # deprecated, please migrate to encodeToAddress to generate the new ckb2021 full 
  # format address as soon as possible
  # Input ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq0rjr2tndx8vdlvspuehkhkgjhxyhxmjgsqw8hsk's password for sign messge by ckb-cli:
  # deploy success, txHash:  
  # 0x7507b59b5f715c0fcd83d55df5569d72ff1cb05cd399afe1f366e03918827a58
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+1. `kuai contract deploy --name [contract-name] --from [0x.....] --export [transaction json file]`
+
+```bash
+$ kuai contract deploy --name always-success --from ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --export ./tx.json
+
+[warn] ANYONE_CAN_PAY script info not found in config!
+[warn] OMNILOCK script info not found in config!
+[warn] `config` changed, regenerate lockScriptInfos!
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  $ kuai contract deploy --name always-success --from ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --export ./tx.json

  # [warn] ANYONE_CAN_PAY script info not found in config!
  # [warn] OMNILOCK script info not found in config!
  # [warn] `config` changed, regenerate lockScriptInfos!
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+
+2. `ckb-cli tx sign-inputs --tx-file [transaction json file] --from-account 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 --add-signatures`
+
+```bash
+$ ckb-cli tx sign-inputs --tx-file ./tx.json --from-account 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 --add-signatures
+
+Password:
+- lock-arg: 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922
+  signature: 0xf3af55aa4c9d6923852465a33d625ff2b5489409908259d79d6aa5bf15a407f10ec32cdd9a9ebff837cdccc2850fc533cab8f08a4c701b57e75530c329f56be201
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  $ ckb-cli tx sign-inputs --tx-file ./tx.json --from-account 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 --add-signatures

  # Password:
  # - lock-arg: 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922
  # signature: 0xf3af55aa4c9d6923852465a33d625ff2b5489409908259d79d6aa5bf15a407f10ec32cdd9a9ebff837cdccc2850fc533cab8f08a4c701b57e75530c329f56be201
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+
+3. `ckb-cli tx send --tx-file [transaction json file]`
+
+```bash
+$ ckb-cli tx send --tx-file ./tx.json
+
+0xfd79d9c3e562b2c5c51df39e49fbb906c63cf2e33e5dbe2bbff12345fa4e6c72
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  $ ckb-cli tx send --tx-file ./tx.json

  # 0xfd79d9c3e562b2c5c51df39e49fbb906c63cf2e33e5dbe2bbff12345fa4e6c72
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+1. `kuai contract deploy --name [contract-name] --from multisig [R] [M] [...args] --export [transaction json file]`
+
+```bash
+$ kuai contract deploy --name always-success  --from multisig 0 2 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e --export ./tx.json
+
+[warn] ANYONE_CAN_PAY script info not found in config!
+[warn] OMNILOCK script info not found in config!
+[warn] `config` changed, regenerate lockScriptInfos!
+The address format generated by generateAddress or scriptToAddress will be deprecated, please migrate to encodeToAddress to generate the new ckb2021 full format address as soon as possible
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  # $ kuai contract deploy --name always-success  --from multisig 0 2 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e --export ./tx.json

  # [warn] ANYONE_CAN_PAY script info not found in config!
  # [warn] OMNILOCK script info not found in config!
  # [warn] `config` changed, regenerate lockScriptInfos!
  # The address format generated by generateAddress or scriptToAddress will be deprecated, please migrate to encodeToAddress to generate the new ckb2021 full format address as soon as possible
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+2. Sign transactions according to multi-signature address: `ckb-cli tx sign-inputs --tx-file [transaction json file] --from-account [args] --add-signatures`
+
+```bash
+# sign by 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922
+$ ckb-cli tx sign-inputs --tx-file ./tx.json --from-account 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 --add-signatures
+
+Password:
+- lock-arg: 0x9ab40977fbad98c42cb0688cb10557a76f32c7ff
+  signature: 0x2395d9d3fde764a6ba78b26699dab410d3609c0303c436768d24add978a64a8c4c4653051fe88124e9f2acb139179f291a40f79ebad93dac3433ad10b39415dd00
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  # sign by 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922
  $ ckb-cli tx sign-inputs --tx-file ./tx.json --from-account 
  # 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 --add-signatures

  # Password:
  # - lock-arg: 0x9ab40977fbad98c42cb0688cb10557a76f32c7ff
  #   signature: 0x2395d9d3fde764a6ba78b26699dab410d3609c0303c436768d24add978a64a8c4c4653051fe88124e9f2acb139179f291a40f79ebad93dac3433ad10b39415dd00
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+
+```bash
+# sign by 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e
+$ ckb-cli tx sign-inputs --tx-file ./tx.json --from-account 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e --add-signatures
+
+Password:
+- lock-arg: 0x9ab40977fbad98c42cb0688cb10557a76f32c7ff
+  signature: 0x6efc1f6359b34fc6bc7edb32059947b61b9914272bda2ed7a7b1aeb0d7d272db7121fcab8cbe25ef8947a0ab1019dcaae3c5f83a9c0f6f486a81fe9787bc16cd01
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  # sign by 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e
  $ ckb-cli tx sign-inputs --tx-file ./tx.json --from-account 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e --add-signatures

  # Password:
  # - lock-arg: 0x9ab40977fbad98c42cb0688cb10557a76f32c7ff
  #   signature: 0x6efc1f6359b34fc6bc7edb32059947b61b9914272bda2ed7a7b1aeb0d7d272db7121fcab8cbe25ef8947a0ab1019dcaae3c5f83a9c0f6f486a81fe9787bc16cd01
```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+
+4. `ckb-cli tx send --tx-file [transaction json file]`
+
+```bash
+$ ckb-cli tx send --tx-file ./tx.json
+
+0xa0fad2950093488baf41b25f3459482b5c7870b21f8d2bb709a731417cb99e33
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```bash
  $ ckb-cli tx send --tx-file ./tx.json

  # 0xa0fad2950093488baf41b25f3459482b5c7870b21f8d2bb709a731417cb99e33
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+
+1. Copy all of this directory directly into the kuai's convention directory where the contracts are placed, which by default should be`/contract/contracts/`
+
+2. Edit the `/contract/cargo.toml` file to add information about the contract to `[workspace]`, e.g.
+
+```toml
+[workspace]
+members = ["tests", "contracts/kuai-mvp-contract", "types", "contracts/always-success"]
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```toml
  [workspace]
  members = ["tests", "contracts/kuai-mvp-contract", "types", "contracts/always-success"]
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+
+```typescript
+networks: {
+    [networkname: string]: {
+        rpcUrl: string
+        prefix: string
+        scripts?: Record<string, ScriptConfig>
+    }
+}
+```
```
Comment (Keith-CY) [inline docs/tutorials/kuai-cli-deploy-contract.md]:
```suggestion
  ```typescript
  networks: {
      [networkname: string]: {
          rpcUrl: string
          prefix: string
          scripts?: Record<string, ScriptConfig>
      }
  }
  ```
```

File: docs/tutorials/kuai-cli-deploy-contract.md (hunk 3 above)
Comment (Keith-CY) [reply docs/tutorials/kuai-cli-deploy-contract.md]: This update makes indention correct, and `#` makes `outputs` commented out so users can copy-paste it directly without deleting the `outputs` in the terminal

Comment (Keith-CY) [reply docs/tutorials/kuai-cli-deploy-contract.md]: @PainterPuppets please check these suggestions

**Comment by PainterPuppets** [comment]
Modified, learned a lot, thanks!

**Pull request merged**